h/zos.h

```diff
@@ -1432,6 +1479,9 @@ bool isCallerSRB(void);
  */
 bool isCallerCrossMemory(void);
 
+int isLowerCasePasswordAllowed();
```

Comment:
Apologies, I just realized that `httpserver.c` is supposed to be buildable on nix/Win, so this change breaks that, because `zos.c` isn't available there.